contracts/LinniaRecords.sol

```diff
@@ -67,6 +68,7 @@ contract LinniaRecords is Ownable {
         bytes32 dataHash, address owner, address attestator,
         string metadata, string dataUri)
         onlyOwner
+        whenNotPaused
```

Comment:
not sure if needed

Reply:
This also makes it easier for many admins to stay in sync.  So an admin and backup admin have less to coordinate.   
Pausing should result in no big state changes such as adding records.  

The admin will have to unpause if they want to call this method.  Not really a big deal.